templates/taxi/car_list.html

```diff
@@ -2,15 +1,32 @@
+{% extends 'base.html' %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="float: right;" href="{% url 'taxi:car-create' %}">+</a></h1>
   {% if car_list %}
-    <ul>
+    <table class="table">
+      <tr>
+        <th>Id</th>
+        <th>Model</th>
+        <th>Manufacturer</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
+        <tr>
+          <td>{{ car.id }}</td>
+          <td>
+            <a href="{% url 'taxi:car-detail' pk=car.id %}">{{ car.model }}</a>
```

Comment:
Ensure that the URL pattern `taxi:car-detail` is correctly defined in your `urls.py` file, as it is used here to link to the car detail page.